Libraries/StyleSheet/processTextDecorationLine.android.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule processTextDecorationLine
+ */
+'use strict';
+
+const textDecorationLineValues = {
```

Comment:
Add a comment that these are bit flags (or reflect that in the variable name) so that someone doesn't add `overline: 3`.